consensus/mint-client/src/config.rs

```diff
@@ -230,6 +230,27 @@ impl MintTxParams {
 }
 #[derive(Subcommand)]
 pub enum Commands {
+    /// Verify whether a signature over given contents verifies with the
+    /// provided pubkey.
+    #[clap(arg_required_else_help = true)]
+    CheckSig {
+        /// The signature to verify.
+        #[clap(
+            long = "signature",
+            use_value_delimiter = true,
+            parse(try_from_str = load_or_parse_ed25519_signature), env = "MC_MINTING_SIGNATURE"
+        )]
+        signature: Ed25519Signature,
+
+        /// The contents that were signed.
+        #[clap(long, parse(try_from_str = FromHex::from_hex), env = "MC_MINTING_CONTENTS")]
```

Comment:
This appears to actually be the hash/digest that are being signed, not the actual contents of the tx file. Whats the actual flow this is intended to be used in?

Reply:
Added the steps to create each field in the comments